js/stateless.js/src/programs/compressed-pda.ts

```diff
@@ -143,6 +225,32 @@ export class LightSystemProgram {
         return outputCompressedAccounts;
     }
 
+    static createDecompressOutputState(
+        inputCompressedAccounts: CompressedAccountWithMerkleContext[],
+        lamports: number | BN,
+    ): CompressedAccount[] {
+        lamports = bn(lamports);
+        const inputLamports = sumupLamports(inputCompressedAccounts);
```

Comment:
stumbled over the sumupLamports name, what about just sumLamports?

Reply:
yea good catch i forgot to capitalize U. grammatically correct is sumUpLamports not sumupLamports nor sumLamports. fixed that.